file_paths.md

```diff
@@ -1,24 +1,341 @@
+            lib                           # :libdir
+            facts                         # used to generate :factpath
+            puppet-module                 # :module_working_dir
+            reports                       # :reportdir
+            server_data                   # :server_datadir
+            state                         # :statedir
+            yaml                          # :yamldir
+        include *
+            facter *
+            openssl *
+        lib *
+            libaugeas.so *
+            libcrypto.so *
+            libfacter.so *
+            libruby.so *
+            libssl.so *
+            ruby *
+                vendor_ruby *             # ruby code
+                    facter.rb *
+                    hiera.rb *
+                    puppet.rb *
+            virt-what *
+                virt-what-cpuid-helper *
+        modules *                         # system modulepath
+        share *
+            augeas *
+            man *
+            vim *
+            locale *
+                config.yaml *             # configuration file used by the gettext-setup gem
+                <lang_COUNTRY> *
+                    LC_MESSAGES *
+                        <project>.mo *    # One directory for each supported locale,
+                                          # containing the MO files for each project.
+                                          # Example for English:
+                                          # /opt/puppetlabs/puppet/share/locale/en_US/LC_MESSAGES/puppet.mo
+        ssl *
+        vendor_modules *                  # :vendormoduledir
+        VERSION                           # puppet-agent package version
+
+    /opt/puppetlabs/pxp-agent *
+        modules *
+            pxp-module-puppet *
+        spool *                            # directory containing results of pxp-agent modules
+        tasks-cache *                      # directory containing a cache of files for all downloaded tasks
+
+    /var/log/puppetlabs *
+        puppet *                          # :logdir                      /var/lib/puppet/log
+            puppet.log                    # not enabled by default
+        pxp-agent *
+            pxp-agent.log                 # enabled by default
+            pcp-access.log                # not enabled by default
+
+    /var/run/puppetlabs *                 # :rundir                      /var/lib/puppet/run
+        agent.pid                         # :pidfile
+        pxp-agent.pid
+
+# puppet-agent 6 (Windows)
+
+On Windows:
+
+- The installation path defaults to `C:\Program Files\Puppet Labs\Puppet`
+    - When installing puppet-agent 32-bit on 64-bit Windows, the default
+      installation path is `C:\Program Files (x86)\Puppet Labs\Puppet`
+- The common app data directory defaults to `C:\ProgramData\PuppetLabs`
+
+The package will create the following services, all running as
+`LocalSystem`, by default:
```

Comment:
nit: Do you add links here as to how to change it? https://puppet.com/docs/puppet/latest/install_windows.html